x/msgfees/types/msgs.go

```diff
@@ -50,9 +55,29 @@ func (msg MsgAssessCustomMsgFeeRequest) ValidateBasic() error {
 	if !msg.Amount.IsPositive() {
 		return errors.New("amount must be greater than zero")
 	}
+	_, err = msg.GetBips()
+	if err != nil {
```

Comment:
nit: since the first argument is never used you can combine this into a one liner. Same goes for the other occurrences in the method where the first arg is never used.

```suggestion
	if _, err := msg.GetBips(); err != nil {
```

Reply:
True.  I changed the others in the validate basic as well to fit the same pattern.